apps/condo/domains/news/components/RecipientCounter.tsx

```diff
@@ -0,0 +1,195 @@
+import { QuestionCircleOutlined } from '@ant-design/icons'
+import { BuildingSection, NewsItemScope, Property as PropertyType, Resident as ResidentType } from '@app/condo/schema'
+import styled from '@emotion/styled'
```

Comment:
Please don't use `emotion`. It's better to use Ant's Col&Row for positioning. See the comment below.

Reply:
I've tried to use `Row` and `Col`, — it has issues with alignment.
Also, I've tried to use a structure `Space > Counter`, — its impossible to justify evenly a list of `Counter` with `Space`, because it does not have `justify-content: space-evenly` prop.